src/main/kotlin/com/dumbdogdiner/stickysurvival/listener/GameEventsListener.kt

```diff
@@ -328,4 +331,43 @@ object GameEventsListener : Listener {
             return
         }
     }
+
+    /**
+     * Bukkit event handler to supplement GameTriggers
+     * Prevents WorldEdit commands (eg. //wand, /tool, //pos1, etc) from running for players in-game
+     */
+    @EventHandler
+    fun onPlayerCommandPreprocess(event: PlayerCommandPreprocessEvent) {
+        val game = event.player.world.game ?: return
+        val pluginManager = Bukkit.getServer().pluginManager
+
+        // Check if the player is in-game && WorldEdit is enabled
+        if (
+            event.player in game.tributesComponent &&
+            pluginManager.isPluginEnabled("WorldEdit")
+        ) {
+            // Get the WorldEdit plugin class
+            val worldEditPlugin = pluginManager.getPlugin("WorldEdit") as WorldEditPlugin
+
+            // CommandManager command names are strings without the first '/'
+            // eg. '//wand' in-game becomes '/wand'
+            // eg. '/worldedit' in-game becomes 'worldedit'
+            val commandManager = worldEditPlugin.worldEdit.platformManager.platformCommandManager.commandManager
+
+            // Get the first section of the command being called
+            val cmdMsg = event.message.split(" ")[0]
+
+            // Strip the first character (the '/') from the string
+            // Now the cmd val matches the same format as WorldEdit's CommandManager! (see above comment)
+            val cmd = cmdMsg.substring(1)
+
+            // Check if the command exists in the (WorldEdit) Command Manager
+            // aka. check if the command was a WorldEdit command
+            if (commandManager.containsCommand(cmd)) {
+                // If so, log to console and cancel the event so the command doesn't run.
+                StickySurvival.instance.logger.info("${event.player.name} tried to issue WorldEdit command '${event.message}' but is in-game!")
```

Comment:
You can use `info` for this (defined [here](https://github.com/DumbDogDiner/StickySurvival/blob/master/src/main/kotlin/com/dumbdogdiner/stickysurvival/util/Logging.kt#L23)) :3